games-simulation/thepowdertoy/thepowdertoy-91.5.330.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A free physics sandbox game"
+DESCRIPTION="The Powder Toy is a free physics sandbox game,\
+which simulates air pressure and velocity, heat, gravity and \
+a countless number of interactions between different substances! \
+The game provides you with various building materials, liquids, gases \
+and electronic components which can be used to construct complex machines,\
+guns, bombs, realistic terrains and almost anything else."
+HOMEPAGE="http://powdertoy.co.uk"
+COPYRIGHT="2008-2010 Stanislaw Skowronek
+	2010 Simon Robertshaw"
```

Comment:
You may wish to change HOMEPAGE and COPYRIGHT to:
```
HOMEPAGE="http://powdertoy.co.uk/"
COPYRIGHT="2008-2010 Stanislaw Skowronek
	2010-2016 Simon Robertshaw"
```
(Changes suggested: add a trailing slash to HOMEPAGE and update copyright years for Simon.)

BTW, I had never heard about this simulator (and have not tried it yet) but it looks great, so thanks for having created a recipe for it!